gradle-consistent-versions-idea-plugin/src/test/java/com/palantir/gradle/versions/intellij/VersionPropsLexerTests.java

```diff
@@ -90,27 +69,78 @@ private static void deleteDirectoryRecursively(Path path) throws IOException {
                 "has.spaces.around : colon = 1",
                 "has.no.spaces.around:equals=1",
                 "has.loads.of.spaces:around   =   equals",
-                "    # comment after spaces"
-            })
+                "    # comment after spaces"));
+    }
+
     @ParameterizedTest
-    public void test_psi_tree_structure(String input) throws Exception {
+    @MethodSource("provideTestCases")
+    public void test_psi_tree_structure(Set<String> input) throws Exception {
```

Comment:
If it's taking a `Set<String>`, at this point there's not point having this be a `@ParameterisedTest` as it only takes one input (I was suggesting have another, separate `@Test` to check to for tests that need deleting). This is also fine too, just turn this `@ParameterisedTest` into a plain `@Test` and have the test cases defined in the test body.